prac_06/guitars.py

```diff
@@ -0,0 +1,41 @@
+"""
+CP1404 2022 Prac 06
+Erica Finlay
+
+Do-from-scratch Exercises
+Guitars!
+
+The solutions branch in GitHub was consulted when completing this activity:
+https://github.com/CP1404/Practicals/tree/solutions/prac_06
+"""
+
+from prac_06.guitar import Guitar
+
+
+def main():
+    """Take user input of guitar name, year & cost and then display in formatted table."""
+    guitars = []
+    print("My guitars!")
+    name = input("Name: ")
+    while name != "":
+        year = int(input("Year: "))
+        cost = float(input("Cost: $"))
+        new_guitar = Guitar(name, year, cost)
+        guitars.append(new_guitar)
+        print(new_guitar, "added to file.")
+        name = input("Name: ")
+    guitars.append(Guitar("Gibson L-5 CES", 1922, 16035.40))
+    guitars.append(Guitar("Line 6 JTV-59", 2010, 1512.9))
+    if guitars:
+        guitars.sort()
+        print("My guitars:")
+        for i, guitar in enumerate(guitars, 1):
+            vintage_string = ""
+            if guitar.is_vintage():
+                vintage_string = " (vintage)"
+            print("Guitar {0}: {1.name:>20} ({1.year}), worth ${1.cost:10,.2f}{2}".format(i, guitar, vintage_string))
+    else:
+        print("No guitars :( Quick go and buy one!")
```

Comment:
Maybe add line breaks within this block of code to improve readability? For example before the If statement

Reply:
Thanks so much, Anthony.  Much appreciate the comments.